src/js/player.js

```diff
@@ -636,11 +636,12 @@ class Player extends Component {
     tag.setAttribute('tabindex', '-1');
     attrs.tabindex = '-1';
 
-    // Workaround for #4583 (JAWS+IE doesn't announce BPB or play button)
+    // Workaround for #4583 (JAWS+IE doesn't announce BPB or play button), and
+    // for the same issue with Chrome (on Windows) with JAWS.
     // See https://github.com/FreedomScientific/VFO-standards-support/issues/78
     // Note that we can't detect if JAWS is being used, but this ARIA attribute
-    //  doesn't change behavior of IE11 if JAWS is not being used
-    if (IE_VERSION) {
+    //  doesn't change behavior of IE11 or Chrome if JAWS is not being used
```

Comment:
should this read "if JAWS is being used" rather than "if JAWS is not being used"?

Reply:
No, the comment is correct. Adding the ARIA role attribute doesn't affect IE11 or Chrome if JAWS isn't being used, so it doesn't matter that we can't detect whether JAWS is being used or not.